monitoring/atproxy/handling.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
+import logging
+import time
+from typing import Tuple
+import uuid
+
+import flask
+
+from monitoring.monitorlib.typing import ImplicitDict
+from .database import db, Query, QueryState
+
+
+def fulfill_query(req: ImplicitDict, logger: logging.Logger) -> Tuple[str, int]:
+    """Fulfill an incoming automated testing query.
+
+    :param req: Request descriptor from requests.py.
+    :param logger: Flask logger.
+    :return: Flask endpoint handler result (content, HTTP code).
+    """
+    t_start = datetime.utcnow()
+    query = Query(type=req.request_type_name(), request=req)
+    timeout = timedelta(seconds=59)
```

Comment:
Manual testing showed that Gunicorn default timeout of 30 seconds interferes with this.
[`--timeout [SEC]` may be added to start.sh to align timeouts.](https://docs.gunicorn.org/en/stable/settings.html#timeout)

Reply:
Thanks; added `--timeout 60` to start.sh